vizro-core/src/vizro/models/_controls/filter.py

```diff
@@ -199,3 +144,98 @@ def _set_actions(self):
                     function=_filter(filter_column=self.column, targets=self.targets, filter_function=filter_function),
                 )
             ]
+
+    def __call__(self, **kwargs):
```

Comment:
This isn't used anywhere yet but will be soon! As with `Figure`, we use `build` for build time and then `__call__`  during dashboard runtime.